vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerLocaleIT.java

```diff
@@ -212,4 +239,30 @@ private void assertNoWarnings() {
                     logEntry));
         }
     }
+
+    private void submitValue(String value) {
```

Comment:
The other tests in this file just this to set the input value, maybe that's good enough?
https://github.com/vaadin/flow-components/blob/5daac2e5276271bb7776aab52ea5546124f01f13/vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerLocaleIT.java#L68-L69

Reply:
Addressed in the latest commits